symengine/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 add_subdirectory(lib)
+add_subdirectory(tests)
 
 set(PY_PATH ${PYTHON_INSTALL_PATH}/symengine)
-install(FILES __init__.py utilities.py compatibility.py
-    DESTINATION ${PY_PATH}
-    )
+FILE(GLOB files "${symengine_SOURCE_DIR}/symengine/*.py")
```

Comment:
I don't like this GLOB thing. It goes against slide 46 in http://www.slideshare.net/DanielPfeifer1/cmake-48475415, but I didn't like it before I read that either. One should explicitly list all the files.